src/toil/job.py

```diff
@@ -714,7 +714,7 @@ class JobDescription(Requirer):
     """
     Stores all the information that the Toil Leader ever needs to know about a Job.
 
-    (requirements information, dependency information, commands to issue,
+    (requirements information, dependency information, pickled job body to run,
```

Comment:
I know that this wasn't your comment, but "requirements" and "dependencies" are kind of synonyms.

Could the contents of "pickled job body" be clearly described here?